addons/sale_timesheet/models/account.py

```diff
@@ -71,9 +71,11 @@ def _check_can_write(self, values):
     def _timesheet_preprocess(self, values):
         values = super(AccountAnalyticLine, self)._timesheet_preprocess(values)
         # task implies so line (at create)
-        if 'task_id' in values and not values.get('so_line') and values.get('employee_id'):
+        if 'task_id' in values and not values.get('so_line') and (values.get('employee_id') or len(self.mapped('employee_id')) > 0):
```

Comment:
`len` is not necessary here
```suggestion
        if 'task_id' in values and not values.get('so_line') and (values.get('employee_id') or self.mapped('employee_id')):
```

Reply:
Sorry, old java reflex ;-)